config/applications.json

```
@@ -2942,5 +2942,13 @@
 		"description": "Fork - a fast and friendly git client.",
 		"link": "https://git-fork.com/",
 		"winget": "Fork.Fork"
+	},
+	"XnConvert": {
+		"category": "Utilities",
+		"choco": "xnview",
+		"content": "xnconvert.install",
```

```suggestion
		"content": "XnConvert",
```
@SHR1SHAK I think using the Application Name would be better.